src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -1353,7 +1353,7 @@ int SketchObject::diagnoseAdditionalConstraints(
     return lastDoF;
 }
 
-int SketchObject::moveGeometries(std::vector<GeoElementId> geoEltIds, const Base::Vector3d& toPoint, bool relative,
+int SketchObject::moveGeometries(std::vector<GeoElementId>& geoEltIds, const Base::Vector3d& toPoint, bool relative,

```

Comment:
Shouldn't it be const reference? Because this will change semantics of that function quite much. With copy you can edit the argument and it will not leak outside function - with reference it will. So we have two options here:

1. It does not do changes to that argument - hence it can be const.
2. It does changes to that argument - in that case it really should be in its own PR and discussed proeprly with explanation why such change is possible. I'd in general avoid things like this, as mutable references are code smell in general.

```suggestion
int SketchObject::moveGeometries(const std::vector<GeoElementId>& geoEltIds, const Base::Vector3d& toPoint, bool relative,
```